### PR TITLE
🛠️ Fix ➾ Add instructions for Updating `taq` Binary to Release Notes

### PR DESCRIPTION
**Pull request by hu3man**
# 🌮 Taqueria PR

## 🪁 Description

Add instructions for users to install the Taqueria v0.22.2 binary 

## 🪂 Pre-Merge Checklist (Definition of Done)

🚦 Required to merge:

- [x] ⛱️ I have completed this PR template in full and updated the title appropriately
- [x] ⛵ My code builds cleanly, and I have manually tested the changes
- [x] 🏖️ New and existing unit tests pass locally and in CI
- [x] 🤿 Corresponding changes have been made to all documentation

## 🎢 Test Plan

N/A

## 🛸 Type of Change

- [ ] 🧽 Chore ➾
- [x] 🛠️ Fix ➾
- [ ] ✨ Feature ➾
- [ ] 👷 Refactor ➾
- [ ] 🧪 Pre-Release ➾
- [ ] 🚀 Release ➾

### CODE REVIEW

File: website/docs/release-notes.mdx [hunk 1]
```diff
@@ -82,6 +82,33 @@ A demonstration dApp has been created to showcase how this works: [https://githu
 
 ### Migrating from Legacy Versions
 
+To upgrade, you must downloade Taqueria v0.22.2 binary, replacing the legacy version
```
Comment (alexzbusko) [inline website/docs/release-notes.mdx:85]: Typo in `download`

File: website/docs/release-notes.mdx [hunk 2]
```diff
@@ -82,6 +82,33 @@ A demonstration dApp has been created to showcase how this works: [https://githu
...
+
+2) Remove the existing `taq` binary
+
+```shell
+rm taq
+```
+
+3) Download the appropriate Taqueria v0.22.2 binary for your operating system
+
+```shell
```
Comment (alexzbusko) [inline website/docs/release-notes.mdx:103]: Same thing with this command, if they are not a super user/use sudo this command will fail

**Review by jchenche — APPROVED** [review]

File: website/docs/release-notes.mdx [hunk 3]
```diff
@@ -82,6 +82,33 @@ A demonstration dApp has been created to showcase how this works: [https://githu
 
 ### Migrating from Legacy Versions
 
+To upgrade, you must downloade Taqueria v0.22.2 binary, replacing the legacy version
+
+This can be done by following these steps:
+
+1) Change into the installation directory
```
[the remark this answers is not in the document]
Comment (hu3man) [reply website/docs/release-notes.mdx:89]: Good call

File: website/docs/release-notes.mdx (hunk 2 above)
Comment (hu3man) [reply website/docs/release-notes.mdx:103]: Yep, thanks!

**Pull request merged**